src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -1272,38 +1139,12 @@ public function testSubmitMultipleExpandedObjectChoices()
         $this->assertNull($form[4]->getViewData());
     }
 
-    public function testSubmitMultipleExpandedNumericChoices()
```

Comment:
in fact, it is already tagged as legacy in 2.8, but this may have been lost during a merge...